src/style.scss

```diff
@@ -9,24 +9,57 @@
 
 @import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/breakpoints";
 @import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/core";
+// essentials setup baseline primitive selectors for any UI development
+@import "./node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariableMap";
```

Comment:
I don't think that this is being used anymore. I removed the references in the Sass and the tooltip bubble didn't change? 

Lines 13 - 16

Reply:
Removing this breaks the insetUtility on my machine, are you seeing the same behavior?

Reply:
Removed code. UI not effected. Committed and pushed.